test/emulator_backend.go

```diff
@@ -179,7 +179,15 @@ func (e *EmulatorBackend) RunScript(
 		}
 	}
 
-	value, err := runtime.ImportValue(inter, interpreter.EmptyLocationRange, e.stdlibHandler, result.Value, nil)
+	staticType := runtime.ImportType(inter, result.Value.MeteredType(nil))
```

Comment:
```suggestion
	staticType := runtime.ImportType(inter, result.Value.Type())
```

Reply:
Nice :+1: 
Updated in https://github.com/onflow/cadence-tools/pull/151/commits/3ee9b4a7aadc2a07e5d224c9fae129eccdaa9c7f